typescript/cli/src/rebalancer/utils/loggerUtils.ts

```diff
@@ -0,0 +1,9 @@
+import { rootLogger } from '@hyperlane-xyz/utils';
+
+export const monitorLogger = rootLogger.child({ module: 'rebalancer-monitor' });
```

Comment:
Not for this PR:

If we want to customize the `rootLogger` for services running in GCP as I did for warp-monitor recently, initializing a logger on module startup breaks the customization subtly because it may happen before or after we replace the `rootLogger`.

We need the customized `rootLogger` to make SDK logs to conform to infra/CLI conventions. The solution is probably to create child loggers off an infra/CLI root logger instance rather than the one from the SDK.